packages/myst-cli/src/process/notebook.ts

```diff
@@ -109,41 +103,41 @@ export async function processNotebook(
         return acc.concat(blockParent(cell, cellMdast.children));
       }
       if (cell.cell_type === CELL_TYPES.raw) {
-        const cellContent = `\`\`\`\n${ensureString(cell.source)}\n\`\`\``;
-        const cellMdast = parseMyst(session, cellContent, file);
-        return acc.concat(blockParent(cell, cellMdast.children));
+        const raw: Code = {
```

Comment:
changes up the process notebook code to just insert nodes, rather then writing the myst and parsing it.